changelogs/v3/releases.json

```diff
@@ -0,0 +1,187 @@
+[
+  {
+    "version": "3.0.0",
```

Comment:
Remove this file.

Reply:
This was done by someone. Its no longer the latest for the branch.